src/test/java/org/zalando/nakadi/service/SubscriptionTimLagServiceTest.java

```diff
@@ -0,0 +1,111 @@
+package org.zalando.nakadi.service;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.Before;
+import org.junit.Test;
+import org.zalando.nakadi.domain.ConsumedEvent;
+import org.zalando.nakadi.domain.EventTypePartition;
+import org.zalando.nakadi.domain.NakadiCursor;
+import org.zalando.nakadi.domain.PartitionEndStatistics;
+import org.zalando.nakadi.domain.Storage;
+import org.zalando.nakadi.domain.Timeline;
+import org.zalando.nakadi.exceptions.ErrorGettingCursorTimeLagException;
+import org.zalando.nakadi.exceptions.InvalidCursorException;
+import org.zalando.nakadi.exceptions.NakadiException;
+import org.zalando.nakadi.exceptions.runtime.InconsistentStateException;
+import org.zalando.nakadi.repository.EventConsumer;
+import org.zalando.nakadi.service.subscription.SubscriptionTimeLagService;
+import org.zalando.nakadi.service.timeline.TimelineService;
+
+import java.time.Duration;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.hasSize;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class SubscriptionTimLagServiceTest {
```

Comment:
There's a time in `TimLag`.

Reply:
Fixed https://github.com/zalando/nakadi/pull/873/commits/36ff06ad6adb6c505d566d59419174244969d7e7